data/gegno/gegno missions.txt

```diff
@@ -0,0 +1,570 @@
+			
+			label observe
+			`	Cautiously retreating to a safer distance, you keep your eyes and ears peeled for the creature, wondering if and where it might appear again. As if to answer your inquiry, the ground begins to rumble, and off to your left, sand and rubble erupt into the air as the beast twists and turns across the surface. Following a loud bellow, the creature screeches in short bursts, almost like chirps, except much more deeper than that of a bird. As some of the dust clears, you see the Vi breathing intensely as they continue to battle the beast.`
+			choice
+				`	(Assist the Vi.)`
+					goto assist
+			
+			label "return encounter"
+			`You find yourself approaching this desert world once again, wondering what happened to the alien figure you previously encountered. Knowing the world is home to a dangerous indigenous species, you are hesitant on landing, making sure to stay clear of any of those creatures' radar signatures.`
+			choice
+				`	(Go search for the alien.)`
+					goto "search again"
+				`	(Carry on.)`
+					defer
+			
+			label "search again"
+			`	With the coordinates of the alien vessel still in the history of your ship's computer, you set off to find it again. On your flight there, your radar suddenly picks up another ship approaching, and you look up to see the same type of ship you saw before flying at a higher altitude over you in the opposite direction. It must be another pilot belonging to the alien species.`
+			choice
+				`	(Attempt to contact the ship.)`
+				`	(Evade the ship.)`
+			`	The ship continues to fly off behind you until it disappears. It seems that whoever is piloting it isn't concerned with helping the alien on the surface. Eventually, the familiar sunken vessel appears on the horizon and you begin to fly around the area in search for life signatures, hoping that the alien is still alive. Nothing comes up besides the faint signatures of the sand creatures at the edges of your radar. You decide to follow the signatures instead.`
+			`	The signatures keep randomly disappearing and reappearing elsewhere, and appear to lead to only empty space. After checking a few of the signatures, an erratic signature appears on the edge of your radar, just like during your first visit. Approaching the location, you recognize the beast that is swinging about on the surface, and catch glimpses of the alien from before continuing to fight it.`
+			choice
+				`	(Assist the alien in defeating the creature.)`
+					goto help
+	
+	on defer
+		set "First Contact: Gegno Vi: deferred"
+
+
+mission "Visit Quarg in Gegno Space"
+	landing
+	name `Travel to <designated system>`
+	description `Travel to the <designated system> as instructed.`
+	substitutions
+		"<alien>" "alien"
+		"<alien>" "Gegno"
+			has "Giaru Gegno: Quarg Contact: offered"
+		"<Alien>" "Alien"
+		"<Alien>" "Gegno"
+			has "Giaru Gegno: Quarg Contact: offered"
+		"<aliens>" "aliens"
+		"<aliens>" "Gegno"
+			has "Giaru Gegno: Quarg Contact: offered"
```

Comment:
Not sure. I worked this out with Derpy so ask him. He might also change this a bit, who knows.

Reply:
Well, for starters, if <aliens> was a global thing, it'd mean that none of the other aliens in the galaxy could use this substitution...

On the flip side, having it be <gegno> that alternates between gegno and aliens depending on the conditions... That makes a lot of sense as a global thing.

Reply:
I would prefer them to remain mission-specific, since the replacement name is generic and because it's only these first few missions that need them. Only if the replacement anmes were made less generic (e.g. `<gegno: alien>`) and there were more missions that used them then making them global would make sense.